pkg/repo/types.go

```diff
@@ -0,0 +1,62 @@
+package repo
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"time"
+)
+
+const (
+	timeformat = "2006-01-02 15:04:05"
+	writeperm  = 0777
```

Comment:
Why `0777` ?

Reply:
That's an error from my part, it should be `0666`. Thanks for the heads-up!